docs/instrument-traces/java-app.md

```diff
@@ -0,0 +1,68 @@
+# Auto-instrument sample java app for traces
+
+In this tutorial we will go through the steps to auto instrument a java app to send traces to Siglens.
+
+### Prerequisites
+- Siglens instance should be running on localhost with ingest port-4318
+- Java app
+- Open Telemetry jar agent
```

Comment:
Don't you explain how to setup a Java app (if they don't already have one) and the Open Telemetry jar agent below? Why are they listed as prerequisites? Also can you add information on how to make siglens run on port 4318 so that it's as easy as possible for anyone to follow along